assets/js/components/user-input/UserInputPreview.js

```diff
@@ -175,18 +185,25 @@ export default function UserInputPreview( props ) {
 							<Button
 								className="googlesitekit-user-input__buttons--next"
 								onClick={ onSaveClick }
+								disabled={ isScreenLoading }
 							>
 								{ __( 'Save', 'google-site-kit' ) }
+								{ isScreenLoading && (
+									<Spinner isSaving={ isScreenLoading } />
```

Comment:
We have the `SpinnerButton` component for this purposes. Could you please update this component to use it instead?